### DISCUSSION TITLE
Incorrect Location Returned From /points/{point} Endpoint

### DISCUSSION THREAD

**Discussion by jmdelawder** [weather-gov/api#261] [Q&A]
**Describe the bug**
/points/{point} endpoint returns incorrect city/state for a latitude/longitude coordinate.

**To Reproduce**

1. Use ArcGIS /suggest endpoint to suggest locations to an end user
[Suggest Endpoint](https://geocode.arcgis.com/arcgis/rest/services/World/GeocodeServer/suggest?f=json&text=New%20York&maxSuggestions=6&location=%7B%22spatialReference%22%3A%7B%22latestWkid%22%3A3857%2C%22wkid%22%3A102100%7D%2C%22x%22%3A-13225312.103694644%2C%22y%22%3A4032428.0429031784%7D)

2. Using ArcGIS results, user selects "New York, NY, USA", which initiates an API request to the /findAddressCandidates endpoint using the result text and magicKey
[Find Address Endpoint](https://geocode.arcgis.com/arcgis/rest/services/World/GeocodeServer/findAddressCandidates?SingleLine=%22New%20York,%20NY,%20USA%22&magicKey=%22dHA9MCNsb2M9NDcwNzM5NiNsbmc9MzMjcGw9MTc5MDg2NCNsYnM9MTQ6MjQzOTQ5MzE=%22&f=json)

3. Lat/Lon returned is {"x":-74.007139999999936,"y":40.714550000000031}. Subsequently querying the NWS API with /points/40.714,-74.007 returns Hoboken, NJ as the city/state in the request, but should be New York, NY

**Additional context**
This can be replicated by simply executing step 3.

**Comment by scadergit** [comment]
It comes down to user expectation of where the center of a geo name actually is. While our presentation layers (Alerts, Radar, Forecast, etc.) use Esri geolocation services, the API uses imported census data. The census considers the entire boundary for "New York, NY," and not the common user's expectation somewhere within Manhattan. You can see on the map below the center of the entire boundary is out near Brooklyn (red circle/x). The blue "x" is where you queried the API, and the green circles are locations in NJ that are tecnincally closer than the census' center for "New York, NY." 

This is a common problem even with Esri geoservices, and we won't be able to address this anytime soon. If that geo name precision is important, it would be better to use Esri for the names and ignore the API metadata.

![image](https://user-images.githubusercontent.com/17500494/101515199-b6f47c00-3943-11eb-94b5-92f0b4b67640.png)
